shared/src/store/action-reducers/exercise.ts

```diff
@@ -197,6 +255,29 @@ function refreshTransfer(
     });
 }
 
+/**
+ * Prepare a {@link PartialExport} for import.
+ *
+ * This includes resetting UUIDs as this cannot be done in the reducer.
+ * @param partialExport The {@link PartialExport} to prepare.
+ */
+export function preparePartialExportForImport(
+    partialExport: PartialExport
+): PartialExport {
+    const copy = cloneDeepMutable(partialExport);
+    // `patientCategories` don't have an `id`
```

Comment:
They don't, but the categories contain templates and these templates have an id

Reply:
Good catch, added this.